src/BrandedNavBar/MobileMenu.tsx

```diff
@@ -209,10 +208,17 @@ type BaseMobileMenuProps = {
   subtext?: string;
   closeMenu?: Function;
   themeColorObject?: ThemeColorObject;
-  logoSrc?: string;
+  showNulogyLogo?: boolean;
 };
 
-const BaseMobileMenu = ({ menuData, closeMenu, subtext, themeColorObject, logoSrc, ...props }: BaseMobileMenuProps) => (
+const BaseMobileMenu = ({
+  menuData,
+  closeMenu,
+  subtext,
+  themeColorObject,
+  showNulogyLogo,
+  ...props
+}: BaseMobileMenuProps) => (
```

Comment:
Can we use React.FC here to provide the children as a prop?

```suggestion
const BaseMobileMenu: React.FC<BaseMobileMenuProps> = ({
  menuData,
  closeMenu,
  subtext,
  themeColorObject,
  showNulogyLogo,
  ...props
}) => (
```

Reply:
@haideralsh I think that's a fine change, although this code was not written by me, just moved around by me.